wikimedia/wikipedia-ios
# Article description - update terminology

**Fixes Phabricator ticket:** https://phabricator.wikimedia.org/T261237

### Notes
* I wasn't sure whether these changes earned a new localization key, but ultimately decided that just changing the English text will be enough. According to https://www.mediawiki.org/wiki/Localisation#Changing_existing_messages, it will still be marked as outdated on the TranslateWiki side for translators to re-check.

### Test Steps
1. Confirm mentions of "title description" in the app have changed to "article description". These changes can be found in the alert view after tapping the first section edit button, the description editing onboarding modal, the description editing modal,
 and the "Learn more" modal.

### Screenshots/Videos *(if applicable)*
Before:
![Screen Shot 2021-03-10 at 10 05 47 AM](https://user-images.githubusercontent.com/3620196/110693429-08afdb00-81ad-11eb-818d-14e1b57fac71.png)
![Screen Shot 2021-03-10 at 10 05 26 AM](https://user-images.githubusercontent.com/3620196/110693250-d9996980-81ac-11eb-9f37-709282d1a3c6.png)
![Screen Shot 2021-03-10 at 10 05 38 AM](https://user-images.githubusercontent.com/3620196/110693513-22e9b900-81ad-11eb-85bc-3ae5231225a4.png)
![Screen Shot 2021-03-10 at 10 05 55 AM](https://user-images.githubusercontent.com/3620196/110693554-309f3e80-81ad-11eb-8ce8-d989b32dcb27.png)

After:
![Screen Shot 2021-03-10 at 2 17 35 PM](https://user-images.githubusercontent.com/3620196/110693634-47459580-81ad-11eb-96db-9961ad959db8.png)
![Screen Shot 2021-03-10 at 2 16 28 PM](https://user-images.githubusercontent.com/3620196/110693666-4dd40d00-81ad-11eb-8694-c41af975304a.png)
![Screen Shot 2021-03-10 at 2 16 38 PM](https://user-images.githubusercontent.com/3620196/110693691-5298c100-81ad-11eb-983a-d79735f31203.png)
![Screen Shot 2021-03-10 at 2 16 44 PM](https://user-images.githubusercontent.com/3620196/110693715-57f60b80-81ad-11eb-9de4-dfb6701b5776.png)
